ERCS/erc-5791.md

```diff
@@ -29,9 +29,9 @@ The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL
 
 ### Requirements
 
-This approach requires that the physical item must have a chip attached to it that fulfills the following requirements:
+This approach requires that the physical item must have a chip attached to it that should be secure and signal authenticity:
 
-- The chip can securely generate and store an ECDSA secp256k1 asymmetric key pair;
+- The chip can securely generate and store an asymmetric key pair;
 - The chip can sign messages using the private key of the previously-generated asymmetric key pair;
 - The chip exposes the public key; and
 - The private key cannot be extracted
```

Comment:
> - The private key cannot be extracted

I might add here "or duplicated by design"; that may seem implied BUT we in fact have explored features whereby keys may be duplicated from chip to chip (for other non-PBT purposes) without ever being revealed.

"by design" acknowledges that over time extraction may become possible due to changes in the state of the art of chip hacking